musicbrainz/csv/get_genre.py

```diff
@@ -0,0 +1,64 @@
+"""
+Script for retrieving the "genre" dumps.
+"""
+
+import time
+import requests
+from requests.exceptions import HTTPError
+import pandas as pd
+from bs4 import BeautifulSoup
+
+URL = "https://musicbrainz.org/ws/2/genre/all"
+PARAMS = dict(fmt="json", limit="50")
+MAX_REQUEST_RETRIES = 3
+
+data = []
+max_records = requests.get(url=URL, params=PARAMS, timeout=50).json()["genre-count"]
+
+for i in range(0, max_records, 50):
+    PARAMS["offset"] = i
```

Comment:
```suggestion
    PARAMS["offset"] = str(i)
```

Reply:
Why is it a string? Should it be an Int?

Reply:
URL parameters are always strings.